src/stubs/Test.stub

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace {{namespace}};
+
+use App\Http\Controllers\{{modelName}}s\{{modelName}}Controller;
+use App\Models\{{modelName}}s\{{modelName}};
+use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Str;
+use Shamaseen\Repository\Tests\TestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class {{modelName}}Test extends TestCase
+{
+    protected string $modelName = '{{modelName}}';
+    protected string  $routePath = "{{modelName}}s";
+
+    /**
+     * @param string $dataName
+     */
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    {
+       $this->routePath = Str::lower($this->routePath);
+        parent::__construct($name, $data, $dataName);
+    }
+
+    public function setUp(): void
+    {
+        parent::setUp();
+    }
+
+
+    public function testIndex()
+    {
+        Route::get($this->routePath, [{{modelName}}Controller::class, 'index']);
+
+        $response = $this->getJson($this->routePath);
+
+        $this->assertContains($response->getStatusCode(), [
+            Response::HTTP_OK, Response::HTTP_PARTIAL_CONTENT,
+        ]);
+    }
+
+    public function testShow()
+    {
+        Route::get("$this->routePath/{id}", [{{modelName}}Controller::class, 'show']);
+
+        $response = $this->getJson("$this->routePath/1");
+
+        $this->assertContains($response->getStatusCode(), [
+            Response::HTTP_OK,
+        ]);
+    }
+
+    public function testCreate()
+    {
+        Route::get("$this->routePath/create", [{{modelName}}Controller::class, 'create']);
+
+        $response = $this->getJson("$this->routePath/create");
+        $this->assertContains($response->getStatusCode(), [
+            Response::HTTP_NO_CONTENT,
+        ]);
+    }
+
+    public function testStore()
+    {
+        Route::post($this->routePath, [{{modelName}}Controller::class, 'store']);
+
+        $data = [
```

Comment:
this will fail right after the generation, maybe you should leave this data empty and add a todo inside it to indicate that a dev should fill the data?